### ISSUE TITLE
[Im2txt] Using TFRecords for captioning dataset

### ISSUE THREAD

**Issue by CaterinaM** [tensorflow/models#961]
Model: Im2txt
Hi,
I'm trying to perform captioning on the MSCOCO validation set and I was wondering if is it possible to exploit the validation set TFRecord shards generated by the MSCOCO preprocessing script (ex.: val-0000*-of-00008) to do this, since captioning a large set of JPEG files takes too long.
How can I adapt run_inference.py to handle SequenceExamples instead of JPG or PNG files?

**Comment by drpngx** [comment]
@CaterinaM you'll find code as an example [here](https://github.com/tensorflow/tensorflow/issues/5234#issuecomment-256989110). Hope that helps. I'm going to close this is issue as it is more appropriate on stackoverflow, where we monitor all questions with the tag `tensorflow`.

**Comment by CaterinaM** [comment]
Thank you!
